srcpkgs/barrier/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'barrier'
 pkgname=barrier
-version=2.1.1
+version=2.1.2
```

Comment:
Where did you find version 2.1.2 ?
The latest [release](https://github.com/debauchee/barrier/releases) is 2.1.1

Reply:
2.1.2 got removed. I thought I had closed that PR already.